docs/2-getting-started/start-free-with-cloud.md

```diff
@@ -4,179 +4,120 @@ title: Start free with Cloud
 
 # Start Free with Recce Cloud
 
-**Launch Recce in under 2 minutes**. Each following feature provides additional value. The progressive features help you get more value from Recce over time.
+**Get started in 3 simple steps**. After signing up, you'll be guided through connecting your GitHub repository and automating your PR validation workflow.
 
-👉 **[Start Free →](https://cloud.reccehq.com){target="_blank"}**
+👉 **[Start Free →](https://cloud.reccehq.com){target="\_blank"}**
 
-## Model Changes and Impact Analysis
+## Step 1: Connect Your GitHub Repository {#github-integration}
 
-Recce shows what changed between **base** and **current** environments and helps assess potential impact. The most common case is comparing your development branch against your production or main branch to see what your changes will impact. 
+After signing up, you'll enter your default project. The first step is connecting your GitHub organization and repository to enable PR tracking and validation.
 
-You can: 
+### What You'll Get
 
-- Explore with the pre-loaded Jaffle Shop data
-- Upload your metadata (see below)
-- **Skip manual upload go directly to [CI/CD automation](#cicd-automation)** 
+Once connected, you can:
 
-<!-- insert a video -->
+- View all open and draft PRs
+- See PR summaries
+
+### Setup Requirements
+
+- GitHub repository with dbt project
+- Permissions for GitHub App installations
+- (Optional) Active PRs with model changes
+
+### Connection Steps
 
-### Upload Metadata
-- Web interface: Click "Update" on the session you want to update in Recce Cloud.
-    1. Click "Update" in base session to upload baseline metadata
-    2. Click "Update" in current session to upload comparison metadata
-    3. Click "Launch" to compare current against base
-- CLI command:
-```
-recce upload-session --session-id <your_session_id>
-```
-Find your session ID in Recce Cloud web interface when clicking "Update" on any session.
+Follow the guidance to connect your GitHub organization and link your repository:
 
-### Required Files
+![Connect GitHub](../assets/images/2-getting-started/connect_github.png){ width="49%" }
+![Link Repository](../assets/images/2-getting-started/link_repository.png){ width="49%" }
 
-Recce needs `manifest.json` and `catalog.json` from both **base** and **current** environments for comparison.
+Once connected, your PRs will appear in the dashboard with basic change summaries:
```

Comment:
"with basic change summaries:"

do users see the summary automatically or they need to click the "Summarize"